app/javascript/components/public_health/CustomExport.js

```diff
@@ -236,23 +236,42 @@ class CustomExport extends React.Component {
                       {this.props.patient_query.filter?.map((filter, index) => {
                         return (
                           <Badge key={`filter-${index}`} variant="primary" className="mr-1">
-                            {filter.filterOption?.title}: {filter.filterOption?.type === 'boolean' && <span>{filter.value ? 'True' : 'False'}</span>}
+                            {`${filter.filterOption?.title}: `}
+                            {['search', 'select', 'number'].includes(filter.filterOption?.type && <span>{filter.value}</span>)}
+                            {filter.filterOption?.type === 'boolean' && <span>{filter.value ? 'True' : 'False'}</span>}
+                            {filter.filterOption?.type === 'date' && (
+                              <span>
+                                {filter.dateOption === ''
+                                  ? 'blank'
+                                  : `${filter.dateOption} ${
+                                      filter.dateOption === 'within'
+                                        ? moment(filter.value?.start).format('MM/DD/YYYY') + ' and ' + moment(filter.value?.end).format('MM/DD/YYYY')
+                                        : moment(filter.value).format('MM/DD/YYYY')
+                                    }`}
+                              </span>
+                            )}
                             {filter.filterOption?.type === 'relative' && (
                               <span>
                                 {filter.relativeOption === 'custom'
                                   ? `in the ${filter.value?.when} ${filter.value?.number} ${filter.value?.unit}`
                                   : filter.relativeOption}
                               </span>
                             )}
-                            {filter.filterOption?.type === 'date' && (
-                              <span>
-                                {filter.dateOption}{' '}
-                                {filter.dateOption === 'within'
-                                  ? moment(filter.value?.start).format('MM/DD/YYYY') + ' and ' + moment(filter.value?.end).format('MM/DD/YYYY')
-                                  : moment(filter.value).format('MM/DD/YYYY')}
-                              </span>
-                            )}
-                            {!['boolean', 'relative', 'date'].includes(filter.filterOption?.type) && <span>{filter.value}</span>}
+                            {filter.filterOption?.type === 'multi' &&
+                              filter.value
+                                ?.map(
+                                  f =>
+                                    `(${f.name} - ${
```

Comment:
do we want parenthesis around these or can we just do commas? it's not quite intuitive to read

Reply:
yeah I don't think the parenthesis is that intuitive but I'm also not sure what the best way to display it would be. I did run this by Alex and he said it looked fine so maybe we can run any additional ideas by him?

Reply:
no longer an issue